PhysicsTools/PatAlgos/python/producersLayer1/lowPtElectronProducer_cff.py

```diff
@@ -88,5 +89,8 @@
 # Schedule rekeying of seed BDT ValueMaps by reco::GsfElectron for run2_miniAOD_UL
 from Configuration.ProcessModifiers.run2_miniAOD_UL_cff import run2_miniAOD_UL
 from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronSeedValueMaps_cff import rekeyLowPtGsfElectronSeedValueMaps
-makePatLowPtElectronsTask_ = cms.Task(makePatLowPtElectronsTask.copy(),rekeyLowPtGsfElectronSeedValueMaps)
+from RecoEgamma.EgammaElectronProducers.lowPtGsfElectronID_cfi import lowPtGsfElectronID
+makePatLowPtElectronsTask_ = makePatLowPtElectronsTask.copy()
+makePatLowPtElectronsTask_.add(rekeyLowPtGsfElectronSeedValueMaps)
+makePatLowPtElectronsTask_.add(lowPtGsfElectronID)
 run2_miniAOD_UL.toReplaceWith(makePatLowPtElectronsTask,makePatLowPtElectronsTask_)
```

Comment:
```suggestion
run2_miniAOD_UL.toReplaceWith(makePatLowPtElectronsTask, cms.Task(makePatLowPtElectronsTask.copy(),
        rekeyLowPtGsfElectronSeedValueMaps,lowPtGsfElectronID))
```
more compact (IIRC, this overrides another comment in this file already)

Reply:
Sorry, I missed this one. I can add it if I update again in response to follow up comments.